test/integration/testLargeResultSet.js

```diff
@@ -113,4 +113,126 @@ describe('Large result Set Tests', function ()
       }
     });
   });
+
+  it('testVariantLarge', function (done)
```

Comment:
could we name this test in more elaborated way? to tell what is the core of the test? is it inserting or selecting?

Reply:
Sure, test is renamed now